### PR TITLE
CSS list-style shorthand.

### PR DESCRIPTION
**Pull request by estelle**
Part of https://github.com/openwebdocs/project/issues/181

* added see also 
* added actual values to the list of values
* moved accessibility down to below examples
* reduced the explanation of an a11y example as the alternative method is the preferred method.
* links in a11y section

### CODE REVIEW

File: files/en-us/web/css/list-style/index.md [hunk 1]
```diff
@@ -175,6 +108,40 @@ List 2
...
+
+```css
+ul {
+  list-style: none;
+}
+
+ul li::before {
+  content: "+ ";
+}
+```
```
Comment (chrisdavidmills) [inline files/en-us/web/css/list-style/index.md:134]: The visual impact here is potentially quite a big issue. 

Question - does something like

```css
li::marker {
  color: transparent;
}
```

hide the bullets without hiding the list from Safari's accessibility tree?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle more great work, thanks! A few comments for you to mull over.

File: files/en-us/web/css/list-style/index.md [hunk 2]
```diff
@@ -50,86 +50,19 @@ list-style: revert-layer;
...
 ```
 
-The `list-style` property is specified as one, two, or three keywords in any order. If {{cssxref("list-style-type")}} and {{cssxref("list-style-image")}} are both set, then `list-style-type` is used as a fallback if the image is unavailable.
+The `list-style` property is specified as one, two, or three keywords in any order. If {{cssxref("list-style-type")}} and {{cssxref("list-style-image")}} are both set, the `list-style-type` is used as a fallback if the image is unavailable.
 
 ### Values
 
 - {{cssxref("list-style-type")}}
-  - : See {{cssxref("list-style-type")}}.
+  - : A `<counter-style>`, {{cssxref("string")}}, or `none`. If omitted in the shorthand, the default `disc` value is used. See {{cssxref("list-style-type")}}.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/list-style/index.md:58]: This is a shorthand property, so the links are just to the long-hand values that make up the shorthand and "none", which maybe shouldn't be there. It, like `symbols()` is a list-style-type value, not a value of list style>

Wondering if I should delete none?

**Comment by estelle** [comment]
@chrisdavidmills Thank you for the review. I addressed the issues and questions you found. thank you. Ready for a re-review with one question: Should we remove none?

File: files/en-us/web/css/list-style/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/css/list-style/index.md:59]: Yeah, I think it is better to leave it, for the reasons you state.

**Review by chrisdavidmills — APPROVED** [review]
@estelle lovely; all comments addressed, and this page is now looking shipshape. Thanks!

**Pull request merged**